.gitignore

```diff
@@ -88,3 +88,5 @@ bower_components
 **/generated
 */generated_tests
 /bin/
+ui/PipelineTest.java
+ui/CameraTest.java
```

Comment:
Gotcha. You should use the explicit paths to avoid accidentally excluding UI tests with the same names.